tests/mocks.py

```diff
@@ -0,0 +1,116 @@
+import datetime
+
+from dateutil.tz import tzlocal
+
+from tests.assets import get_asset
+
+
+class s3ClientMock:
```

Comment:
for future expansion, follow the Only Mock Types You Own principle and introduce an abstraction layer over Boto if `S3Client` gets used by many different tasks

Reply:
refactored which removes the need for mocked responses.